### DISCUSSION TITLE
How to add bootstrap-vcpkg.bat proxy

### DISCUSSION THREAD

**Discussion by bitnick10** [microsoft/vcpkg#18668] [Q&A] [answered]
While calling Windows API function WinHttpSendRequest got error 0x00002EE2: Timeout
I'm stuck at this bootstrap-vcpkg.bat,how to set proxy to let this bat script work?

**Comment by mathisloge** [comment, marked as answer]
Have you set HTTP_PROXY/HTTPS_PROXY ?

https://docs.cloudfoundry.org/cf-cli/http-proxy.html

**Reply by pentatracy5** [reply]
I am using Win10. What should I set for this enviroment variable? Thanks
![image](https://user-images.githubusercontent.com/101581440/158287314-9d0c68e0-8927-4700-961c-a6cc90293689.png)

**Reply by mathisloge** [reply]
your proxy address if you are behind a proxy

**Comment by pentatracy5** [comment]
> While calling Windows API function WinHttpSendRequest got error 0x00002EE2: Timeout I'm stuck at this bootstrap-vcpkg.bat,how to set proxy to let this bat script work?

Has your problem been solved? I have the same one.
![image](https://user-images.githubusercontent.com/101581440/158210215-c59c399c-d846-4b3d-8eaa-2ea09057c8b7.png)

**Reply by pentatracy5** [reply]
Oh sorry. It's not the same. The API I met is WinHttpReadData. Thanks.
